node/benchmarks/relay_server.js

```diff
@@ -76,6 +76,8 @@ function RelayServer(opts) {
     });
     self.relay.handler = ServiceProxy({
         channel: self.relay,
+        totalRpsLimit: 30000,
+        defaultServiceRpsLimit: 30000,
```

Comment:
Might make a comment that this is 50% over our design goal to avoid interfering with the benchmark.